core/src/test/java/org/apache/calcite/util/UtilTest.java

```diff
@@ -914,7 +914,7 @@ private List<Integer> makeConsList(int start, int end) {
     }
     // Example locale names in Locale.toString() javadoc.
     String[] localeNames = {
-        "en", "de_DE", "_GB", "en_US_WIN", "de__POSIX", "fr__MAC"
+        "en", "de_DE", "fr_CA"
```

Comment:
The change implies that we no longer support certain patterns when parsing locales so I think we should mention the change in behavior in `history.md`.

Reply:
added such note